docs/docs/en/guide/installation/pseudo-cluster.md

```diff
@@ -142,7 +142,8 @@ export PATH=$HADOOP_HOME/bin:$SPARK_HOME1/bin:$SPARK_HOME2/bin:$PYTHON_HOME/bin:
 
 ## Initialize the Database
 
-DolphinScheduler metadata is stored in the relational database. Currently, supports PostgreSQL and MySQL. If you use MySQL, you need to manually download [mysql-connector-java driver][mysql] (8.0.16) and move it to the lib directory of DolphinScheduler, which is `tools/libs/`. Let's take MySQL as an example for how to initialize the database:
+DolphinScheduler metadata is stored in the relational database. Currently, supports PostgreSQL and MySQL. If you use MySQL, you need to manually download [mysql-connector-java driver][mysql] (8.0.16) and move it to the libs directory of DolphinScheduler
+which is `api-server/libs/` and `alert-server/libs` and `master-server/libs` and `worker-server/libs`. Let's take MySQL as an example for how to initialize the database:
```

Comment:
The preview desc is correct, because part is **Initialize the Database**. maybe we should a separate part to tell user how to use mysql datasource instead of this part

Reply:
What do you mean how to use mysql datasource?

Reply:
use mysql as meta store for dolphinscheduler

Reply:
I think [how to](https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/guide/howto/general-setting.html ) is a great place to hold this. we could ad new page and link all relation doc to the new one. for example all installation(including container) and datasource center will face the same question. Actually I already add some related doc in https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/guide/datasource/introduction.html